www-apps/otrs/otrs-5.0.18.ebuild

```diff
@@ -0,0 +1,153 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit user systemd
+
+DESCRIPTION="OTRS is an Open source Ticket Request System"
+HOMEPAGE="https://otrs.org/"
```

Comment:
This should rather be `https://www.otrs.com/` otherwise I get an SSL certificate error.